code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -209,16 +217,87 @@
 			playsound(loc, 'sound/voice/dionatalk1.ogg', 50, 0) //Credit https://www.youtube.com/watch?v=ufnvlRjsOTI [0:13 - 0:16]
 			m_type = 2
 
+		if("chirp", "chirps")
+			var/M = handle_emote_param(param)
+			message = "<B>[src]</B> chirps[M ? " at [M]" : ""]."
+			switch(rand(1,10))
+				if(1)
+					playsound(loc, 'sound/effects/voxchirp01.ogg', 50, 1, frequency = get_age_pitch()) //Credit to AlexTriceratops123  (https://www.youtube.com/watch?v=uA5yGyB_z5U) for the sound.
```

Comment:
This code is really gross.

Just make it something like `playsound(loc, 'sound/effects/voxchirp[rand(1,10)].ogg', 50, 1, frequency = get_age_pitch())`

And make adjustments to the naming structure.